torch/_torch_docs.py

```diff
@@ -3587,6 +3587,34 @@ def merge_dicts(*dicts):
              [  2.0774,  -0.8187]]])
 """.format(**common_args))
 
+add_docstr(torch.matrix_exp,
+           r"""
+matrix_power(input) -> Tensor
+
+Returns the matrix exponential. Supports batched input.
+
+Args:
+    {input}
+
+Example::
+
+    >>> a = torch.randn(2, 2, 2)
+    >>> a[0, :, :] = torch.eye(2, 2)
+    >>> a[1, :, :] = 2 * torch.eye(2, 2)
+    >>> a
+    tensor([[[1., 0.],
+             [0., 1.]],
+
+            [[2., 0.],
+             [0., 2.]]])
+    >>> torch.matrix_exp(a)
+    tensor([[[2.7183, 0.0000],
+             [0.0000, 2.7183]],
+
+             [[7.3891, 0.0000],
+              [0.0000, 7.3891]]])
```

Comment:
Can you provide a non-trivial matrix as well?

Reply:
I think this case is just shows what the matrix functions actually is doing. For non-scalar functions it is just not obvious what the result is going to be unless you know the eigenvalues. The non-trivial cases are covered in the tests.

Reply:
Maybe adding a skew-symmetric 2x2 matrix such as `exp( [[0,-a], [a,0]]) =[[cos a, -sin a], [sin a, cos a]]` to show that this is different to `torch.exp`?

Reply:
That is actually a good idea! Thanks!